examples/complete/variables.tf

```diff
@@ -13,13 +13,13 @@ variable "region" {
 variable "prefix" {
   type        = string
   description = "The prefix to apply to all resources created by this example."
-  default     = "event-streams-com"
+  default     = "geretain-sky"
```

Comment:
remove your name from the prefix, and definitely do not use `geretain` or thing will never get nuked.

Reply:
yes @ocofaigh, I just did for testing, it take too long to create one, so did that.